spec/features/user_signs_out_spec.rb

```diff
@@ -10,21 +10,21 @@
   # * When I opt to sign out, I get a confirmation that my identity has been
   #   forgotten on the machine I'm using
 
-  # scenario 'authenticated user signs out' do
-    # user = FactoryGirl.create(:user)
+  scenario 'authenticated user signs out' do
+    user = FactoryGirl.create(:user)
 
-    # visit new_user_session_path
+    visit new_user_session_path
 
-    # fill_in 'Email', with: user.email
-    # fill_in 'Password', with: user.password
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
 
-    # click_button 'Log in'
+    click_button 'Log in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.